chains/ethereum/connection_test.go

```diff
@@ -20,20 +21,19 @@ func TestConnect(t *testing.T) {
 	conn.Close()
 }
 
+//  TestSubscribe makes sure we can subscribe to the results of a streaming filter query
 func TestSubscribe(t *testing.T) {
 	client := ethtest.NewClient(t, TestEndpoint, AliceKp)
-
 	_ = deployTestContracts(t, client, aliceTestConfig.id, AliceKp)
-	conn := newLocalConnection(t, aliceTestConfig)
-	l := NewListener(conn, aliceTestConfig, TestLogger, &blockstore.EmptyStore{}, make(chan int), make(chan error))
-	defer conn.Close()
 
 	q := eth.FilterQuery{}
 
-	_, err := l.conn.subscribeToEvent(q)
+	ch := make(chan ethtypes.Log)
+	sub, err := client.Client.SubscribeFilterLogs(context.Background(), q, ch)
```

Comment:
Is passing context.Background() appropriate here?

Reply:
Yeah. Still wrapping my head around contexts, I think `context.TODO()` would also be appropriate, but since we don't use it for anything these should be equivalent.